pom.xml

```diff
@@ -848,6 +860,16 @@
                 <version>${che.version}</version>
             </dependency>
             <dependency>
+                <groupId>org.flywaydb</groupId>
```

Comment:
I think you should move these dependencies to [che-dependencies](https://github.com/eclipse/che-dependencies/blob/master/pom.xml) project